### PR TITLE
#584 Added my name to credits

### LINKED ISSUE #584: Arnav Rathore - credits
### Description

I want my name in Cyan

### Acceptance Criteria

I want my name on the credits page

### Proposed Solution

add my name to 'CreditsPage.tsx' file.

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by AR619**
## Screenshots

_If you made UI changes you must post a screenshot of the whole page for each change in
 1) 
er ss 1

 
2) the smallest possible window_
er ss 2

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing

- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #584

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+'DATABASE_URL="postgresql://postgres:password@localhost:5432/nerpm?schema=public"'
```
Comment (Peyton-McKee) [inline .env:1]: This file should be deleted. Your env file should be in your backend!

**Review by Peyton-McKee — CHANGES_REQUESTED** [review]
Only one change to your code! Also, In your PR, make sure to delete any topics that you don’t end up filling out. And for your screenshots we only need screenshots of the credits page on the website, not the physical code! Visit localhost:3000/credits and take a screenshot with your name there!

File: .env (hunk 1 above)
Comment (AR619) [reply .env:1]: Hi where exactly can I find this 'DATABASE_URL="postgresql://postgres:password@localhost:5432/nerpm?schema=public"' line? I can't find the folder that it's in

**Review by Peyton-McKee — DISMISSED** [review]
Looks Good Now! One important thing for the future, make sure that your commit messages start with “#<YourIssueNumber>” So instead of saying “yourissuenumber”, for this ticket you should say “#584”, and you should preface every commit message with that.

**Review by RChandler234 — APPROVED** [review]
approved!

**Pull request merged**